AspNetCoreApplication/Repositories/BookCategoryRepository.cs

```diff
@@ -9,17 +11,21 @@
 
 namespace AspNetCoreApplication.Repositories
 {
-    public class BookCategoryRepository : IBookCategoryRepository
+    public class BookCategoryRepository : Repository<Book>, IBookCategoryRepository
```

Comment:
Cái này là không được phép bởi vì em đang khai báo Repository của BookCategory thì không có lý do gì để thêm hết các hàm CRUD của Book cả.
```suggestion
    public class BookCategoryRepository : IBookCategoryRepository
```

Reply:
done